src/explore-education-statistics-admin/src/pages/release/footnotes/ReleaseFootnotesPage.tsx

```diff
@@ -69,17 +85,33 @@ const ReleaseFootnotesPage = ({
     return (
       <>
         {canUpdateRelease && (
-          <ButtonLink
-            to={generatePath<ReleaseRouteParams>(
-              releaseFootnotesCreateRoute.path,
-              {
-                publicationId,
-                releaseId,
-              },
+          <div className="dfe-flex dfe-justify-content--space-between">
+            <ButtonLink
```

Comment:
When we're reordering, this button still shows, which is a little confusing. Would suggest we hide this whilst reordering.